test/api/test_plan_serialization.cpp

```diff
@@ -44,11 +44,13 @@ static void test_helper(string sql, vector<string> fixtures = vector<string>())
 		//		printf("[%d] Optimized plan\n", i);
 
 		BufferedSerializer serializer;
+		serializer.SetVersion(PLAN_SERIALIZATION_VERSION);
 		plan->Serialize(serializer);
 		//		printf("[%d] Serialized plan\n", i);
```

Comment:
```suggestion
```

Reply:
Thanks! I rather not change code that's not part of this PR so prefer to leave this line in place.